src/content/deployment/flavors.md

```diff
@@ -6,162 +6,335 @@ description: >
-
-```json
-{
-  "version": "0.2.0",
-  "configurations": [
-    {
-      "name": "staging",
-      "request": "launch",
-      "type": "dart",
-      "program": "lib/main_development.dart",
-      "args": ["--flavor", "staging", "--target", "lib/main_staging.dart" ]
-    }
-  ],
-  "compounds": []
-}
-```
-
-You can now run the terminal command
-`flutter run --flavor staging` or you can set up a run
-configuration in your IDE.
-
-{% comment %}
-TODO: When available, add an app sample.
-{% endcomment -%}
-
-## Launching your app flavors
-
-1. Once the flavors are set up, modify the Dart code in
-**lib** / **main.dart** to consume the flavors.
-2. Test the setup using `flutter run --flavor staging`
-at the command line, or in your IDE.
-
-For examples of build flavors for [iOS][], [macOS][], and [Android][],
-check out the integration test samples in the [Flutter repo][].
-
-## Retrieving your app's flavor at runtime
-
-From your Dart code, you can use the [`appFlavor`][] API to determine what
-flavor your app was built with.
-
-## Conditionally bundling assets based on flavor
-
-If you aren't familiar with how to add assets to your app, see
-[Adding assets and images][].
-
-If you have assets that are only used in a specific flavor in your app, you can
-configure them to only be bundled into your app when building for that flavor.
-This prevents your app bundle size from being bloated by unused assets.
-
-Here is an example:
-
-```yaml
-flutter:
-  assets:
-    - assets/common/
-    - path: assets/staging/
-      flavors:
-        - staging
-    - path: assets/production/
-      flavors:
-        - production
-```
-
-In this example, files within the `assets/common/` directory will always be bundled
-when app is built during `flutter run` or `flutter build`. Files within the
-`assets/staging/` directory are bundled _only_ when the `--flavor` option is set
-to `staging`. Similarly, files within the `assets/production` directory are
-bundled _only_ if `--flavor` is set to `production`.
+1.  Update `AndroidManifest.xml` in your IDE:
+
+    * In the `flavors_example` project, navigate to
+      `android/app/src/main` and open `AndroidManifest.xml`.
+
+    * Replace the value for `android:label` with
+      `@string/app_name`.
+
+      ```xml title="AndroidManifest.xml"
+      <manifest xmlns:android="http://schemas.android.com/apk/res/android">
+          <application
+            android:label="@string/app_name"
+            ...
+          />
+      />
+      ```
+
+1.  Launch the app for each product flavor (`staging`,
+    `production`) and check to make sure that the
+    app display name has changed for each.
+
+    * To launch a product flavor, see the steps in
+      [Launch a flavor][].
+
+    * In the Android App Emulator, go to the list of apps.
+      You should see one for `Flavors p...` and
+      `Flavors s...`.
+
+    * To see more information for `Flavors p...` or
+      `Flavors s...`, hold down the mouse over one of them
```

Comment:
This wording troubles me. Isn't it more correct to say "hover over one of them and right-click the mouse button to bring up the menu. Select **App info**" or something like that?

Reply:
Thanks for catching that. Just did a quick look around and updated this to match what I'm seeing elsewhere.

To see more information for `Flavors p...` or
`Flavors s...`, long-press the icon for one of them
and and select `App info`.